doc/wiki/How_To.md

```diff
@@ -0,0 +1,111 @@
+# How to?
+## This document should help you as a user to understand all functionalities of the application.
+
+##Introduction
+After logging in you will be navigated to the dashboard screen.
+Here you see the navigation at the left border and some fancy dashboards in the content area.
+They give you some information about children and can be defined by you, but later more about that (Verlinkung section Dashboard).
+In the navigation you can choose between the different sections (Dashboard, Children, Schools, Notes, Attendance Register).
+
+In the following the apps' basic functions are explained step by step.
+
+##Children
+To enter the Children section click on the button in the navigation. You will be navigated to the Children List Screen.
+###... create a child?
+1. On the Children List Screen click on the button with the plus (and the text "Add New") at the upper right side.
+  This will open an empty child view
+
+2. Fill in the needed attributes for the new child.
+After this click "Save" in the top right corner.
+You can see that saving was successful when the child's name is the headline.
+
+### ... edit a child?
+1. On the Children List Screen search for the child you want to edit and click on the corresponding entry in the list.
+It is the best to click on the name or the project number (PN), cause clicking on the school will navigate you to the school's detail page.
```

Comment:
maybe we should consider changing this confusing behaviour? ;-)

Reply:
Sure, create a new issue ;)